src/vmm/src/logger/logging.rs

```diff
@@ -300,17 +300,18 @@ mod tests {
             log::LevelFilter::Error
         );
     }
+
```

Comment:
The commit message:

> levelfilter_from_str_all_variants is a better place to logic.

I cannot parse that sentence :) should it be "a better place to put the logic"?

Reply:
yep, sorry fixed it.